lib/src/screens/memory/memory_screen.dart

```diff
@@ -133,6 +135,8 @@ class MemoryScreen extends StatelessWidget {
                   state.memory.highlightIndices.toList(),
                   state.memory.indexOfSelectedHighlight,
                   MemoryService().updateSelectedHighlight,
+                  NextSearchIndexAction(),
```

Comment:
on every rebuild, aren't the indices getting changed ?

Reply:
It's only invoked on keyboard button press. So the intent has the next value but action won't get invoked unless user presses the button